### DISCUSSION TITLE
Performance when combining scores

### DISCUSSION THREAD

**Discussion by mglev1n** [PGScatalog/pgsc_calc#110] [Q&A]
When using `pgsc_calc` for more than a few scores, the pipeline seems to take a very long time. Despite using the `--parallel` argument and having access to HPC resources, the step that combines scores into a single scoring file seems to run serially (reading score1, performing QC, appending to scoring file; reading score2, performing QC, appending, ...). Is there a more efficient approach for creating a combined scoring file (either using the `pgsc_calc` pipeline, or preparing the combined scoring file prior to running the pipeline)?

**Comment by smlmbrt** [comment]
Thanks @mglev1n, we agree and have opened some issues to explore how to make this step faster. It will coincide with allowing wide scoring files (e.g. multiple weights for a single set of variants) as well.
